packages/core/src/tracing/instrumentation/database/ioredis.js

```diff
@@ -71,7 +99,7 @@ function instrumentSendCommand(original) {
       const span = cls.startSpan(exports.spanName, constants.EXIT);
       span.stack = tracingUtil.getStackTrace(wrappedInternalSendCommand);
       span.data.redis = {
-        connection: `${client.options.host}:${client.options.port}`,
+        connection: clusterConnectionString || `${client.options.host}:${client.options.port}`,
```

Comment:
This is a little bit "hacky" because if a customer uses both single client and cluster connection, the connection string could be wrong. But I would ignore the edge case.

Reply:
That is a likely, unlikely situation. We can ignore it for now.

Reply:
I agree as well. If an issue arises in the future, we can address it then. We know where to make the fix. 😄